ee/frontend/mobile-replay/transform.test.ts

```diff
@@ -0,0 +1,303 @@
+                        screen: 'App Home Page',
+                        width: 300,
+                        height: 600,
+                    },
+                    timestamp: 1,
+                    type: 4,
+                },
+                {
+                    type: 2,
+                    data: {
+                        wireframes: [
+                            {
+                                id: 12345,
+                                x: 11,
+                                y: 12,
+                                width: 100,
+                                height: 30,
+                                // clip: {
+                                //   bottom: 83,
+                                //   right: 44,
+                                // },
+                                type: 'text',
+                                text: 'Ⱏ遲䩞㡛쓯잘ጫ䵤㥦鷁끞鈅毅┌빯湌Თ',
+                                style: {
+                                    // family: '疴ꖻ䖭㋑⁃⻋ꑧٹ㧕Ⓖ',
+                                    // size: 4220431756569966319,
+                                    color: 'red',
+                                },
+                            },
+                            {
+                                id: 12345,
+                                x: 25,
+                                y: 42,
+                                width: 100,
+                                height: 30,
+                                // clip: {
+                                //   bottom: 83,
+                                //   right: 44,
+                                // },
+                                type: 'image',
+                                base64: heartEyesEmojiURL,
+                            },
+                        ],
+                    },
+                    timestamp: 1,
+                },
+            ])
+            expect(exampleWithImage).toMatchSnapshot()
+        })
+
+        test('can convert rect with text', () => {
+            const exampleWithRectAndText = posthogEEModule.mobileReplay?.transformToWeb([
+                {
+                    data: {
+                        width: 300,
+                        height: 600,
+                    },
+                    timestamp: 1,
+                    type: 4,
+                },
+                {
+                    type: 2,
+                    data: {
+                        wireframes: [
+                            {
+                                id: 12345,
+                                x: 11,
+                                y: 12,
+                                width: 100,
+                                height: 30,
+                                type: 'rectangle',
+                                style: {
+                                    color: 'red',
+                                    borderColor: 'blue',
+                                    borderWidth: '4',
+                                    borderRadius: '10px',
```

Comment:
I'd omit `px` here since its always pixels anyway

Reply:
have updated to allow `10` or `10px`